web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
+            <li><a href="{{ pathPrefix }}/config">Config</a></li>
```

Comment:
From daily usage I see from others and myself, it's mostly scrolling down to the targets. Sometimes the rules (they are really hard to look at right now) and the rest really rarely.

So I'm very much against making it targets another hop from the Status page.
Why not just have `Alerts   Graph   Targets   Status   Help`? Perfectly fits into the current navbar.